foundation-extensions/mobile-core/configuration/configuration-api-reference.md

```diff
@@ -465,3 +465,44 @@ MobileCore.configureWithFileInAssets("exampleJSONfile.json");
 {% endtab %}
 {% endtabs %}
 
+## clearUpdatedConfiguration (iOS 3.x.x Only)
+
+You can clear any programmatic updates made to configuration via the `clearUpdatedConfiguration` API. This will clear programmatic updates to configuration made via the `updateConfigurationWith(configDict:)` API. It will also clear any updates to the `PrivacyStatus` made via `setPrivacyStatus(_ status:)`. 
```

Comment:
Same comment here about using the configuration, instead of just saying configuration.